python/ray/worker.py

```diff
@@ -865,6 +924,10 @@ def _process_task(self, task):
         function_name = self.function_execution_info[self.task_driver_id.id()][
             function_id.id()].function_name
 
+        if not self.serialization_context_map.has_key(self.task_driver_id):
+            _initialize_serialization()
+        register_existing_class()
```

Comment:
Why do we need to add this? Shouldn't the regular import mechanism be sufficient?

Reply:
To separate the content for different driver id, we have to do that, otherwise, it is hard to do GC for the context.